.github/workflows/CI.yml

```diff
@@ -28,6 +28,11 @@ jobs:
           - runner: windows-latest
             target: x86_64-pc-windows-msvc
             optional: true
+          - runner: ubuntu-latest
+            target: aarch64-unknown-linux-gnu
+            optional: true
+            build-only: true
```

Comment:
Debatable; If we're going to support another architecture, we might as well test on it as well. That should _in principle_ always be fine, but I wouldn't necessarily disable tests on those.

Reply:
Good point! As the `raspberry` target relies on cross-compilation (as opposed to being compiled on a real hardware), `cargo test` fails (as it runs the real binary).
An option could be to have a `self-hosted` runner here if we want to have `raspberry` as a first-class citizen.

Reply:
Actually looks like [cross](https://github.com/cross-rs/cross) does the heavy lifting here and allows to run `cross-test` (via `docker`). Looks simpler.

Reply:
> As the raspberry target relies on cross-compilation 

Ho, didn't know that :( ... That's a bummer, and indeed, diminishes a lot the value we would get from testing. So I suppose it's fine, but let's capture this as a comment/justification in the workflow configuration. Because I'll forget that conversion tomorrow already.

Reply:
Actually `cross` solves this and tests can be run too. I made the necessary changes in a more recent commit.

I also validated that the generated binary runs fine on a real raspberry when using `musl` and target `aarch64-unknown-linux-musl`. Maybe we should directly use the musl `target` for `CI` too, as it's most probably the one we should use to ship binaries.

Reply:
> validated that the generated binary runs fine on a real raspberry

Yolo. Love it. 

Goal for next month: keep a running Amaru node in the office on your raspberry.